<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GY_GenerateBinFile</class>
 <widget class="QWidget" name="GY_GenerateBinFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>906</width>
           <height>551</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTabWidget" name="tabWidget">
            <property name="tabPosition">
             <enum>QTabWidget::TabPosition::West</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::TabShape::Triangular</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="elideMode">
             <enum>Qt::TextElideMode::ElideNone</enum>
            </property>
            <property name="documentMode">
             <bool>false</bool>
            </property>
            <property name="tabsClosable">
             <bool>false</bool>
            </property>
            <property name="movable">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>动画排序</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_6">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QScrollArea" name="scrollArea_3">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents_3">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>868</width>
                   <height>535</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>信息配置</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_5">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QScrollArea" name="scrollArea_2">
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents_2">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>868</width>
                   <height>535</height>
                  </rect>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="0" column="0">
                   <widget class="QGroupBox" name="groupBox">
                    <property name="maximumSize">
                     <size>
                      <width>220</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>微软雅黑</family>
                      <pointsize>9</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="mouseTracking">
                     <bool>false</bool>
                    </property>
                    <property name="tabletTracking">
                     <bool>false</bool>
                    </property>
                    <property name="acceptDrops">
                     <bool>false</bool>
                    </property>
                    <property name="layoutDirection">
                     <enum>Qt::LayoutDirection::LeftToRight</enum>
                    </property>
                    <property name="autoFillBackground">
                     <bool>false</bool>
                    </property>
                    <property name="title">
                     <string>公有信息</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignmentFlag::AlignCenter</set>
                    </property>
                    <property name="flat">
                     <bool>false</bool>
                    </property>
                    <property name="checkable">
                     <bool>false</bool>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_4">
                     <item row="5" column="0">
                      <layout class="QHBoxLayout" name="horizontalLayout">
                       <item>
                        <widget class="QLabel" name="label_shaftInfo_2">
                         <property name="maximumSize">
                          <size>
                           <width>250</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>轴体选择</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QComboBox" name="comboBox">
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">color: rgb(255, 0, 0);</string>
                         </property>
                         <item>
                          <property name="text">
                           <string>轴体A(默认)</string>
                          </property>
                         </item>
                         <item>
                          <property name="text">
                           <string>轴体B</string>
                          </property>
                         </item>
                         <item>
                          <property name="text">
                           <string>轴体C</string>
                          </property>
                         </item>
                         <item>
                          <property name="text">
                           <string>轴体D</string>
                          </property>
                         </item>
                         <item>
                          <property name="text">
                           <string>轴体E</string>
                          </property>
                         </item>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="7" column="0">
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <widget class="QLabel" name="label_keyMin">
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>按键最小值</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QDoubleSpinBox" name="doubleSpinBox_keyMin">
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="value">
                          <double>0.200000000000000</double>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="1" column="0">
                      <layout class="QHBoxLayout" name="horizontalLayout_6">
                       <item>
                        <widget class="QLabel" name="label_whiteBalance_R">
                         <property name="maximumSize">
                          <size>
                           <width>250</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>白平衡R</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSpinBox" name="spinBox_whiteBalance_R">
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="maximum">
                          <number>255</number>
                         </property>
                         <property name="value">
                          <number>255</number>
                         </property>
                         <property name="displayIntegerBase">
                          <number>10</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="2" column="0">
                      <layout class="QHBoxLayout" name="horizontalLayout_5">
                       <item>
                        <widget class="QLabel" name="label_whiteBalance_G">
                         <property name="maximumSize">
                          <size>
                           <width>250</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>白平衡G</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSpinBox" name="spinBox_whiteBalance_G">
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="maximum">
                          <number>255</number>
                         </property>
                         <property name="value">
                          <number>117</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="0" column="0">
                      <layout class="QHBoxLayout" name="horizontalLayout_7">
                       <item>
                        <widget class="QLabel" name="label_returnSpeed">
                         <property name="maximumSize">
                          <size>
                           <width>250</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>回报率</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSpinBox" name="spinBox_returnSpeed">
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="minimum">
                          <number>1</number>
                         </property>
                         <property name="maximum">
                          <number>2</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="6" column="0">
                      <layout class="QHBoxLayout" name="horizontalLayout_2">
                       <item>
                        <widget class="QLabel" name="label_keyMax">
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>按键最大值</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QDoubleSpinBox" name="doubleSpinBox_keyMax">
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="value">
                          <double>1.200000000000000</double>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="8" column="0">
                      <widget class="QPushButton" name="pushButton_keyMaxorMinValue">
                       <property name="text">
                        <string>确认按键默认值</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="0">
                      <layout class="QHBoxLayout" name="horizontalLayout_4">
                       <item>
                        <widget class="QLabel" name="label_whiteBalance_B">
                         <property name="maximumSize">
                          <size>
                           <width>250</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>白平衡B</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSpinBox" name="spinBox_whiteBalance_B">
                         <property name="font">
                          <font>
                           <family>微软雅黑</family>
                           <pointsize>10</pointsize>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="maximum">
                          <number>255</number>
                         </property>
                         <property name="value">
                          <number>87</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="4" column="0">
                      <widget class="Line" name="line">
                       <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QGroupBox" name="groupBox_2">
                    <property name="title">
                     <string>键轴信息</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignmentFlag::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
